src/d3a/models/strategy/external_strategies/load.py

```diff
@@ -244,6 +244,12 @@ def event_market_cycle_prices(self):
     def _update_bid_aggregator(self, arguments):
         assert set(arguments.keys()) == {'price', 'energy', 'type', 'transaction_id'}
         bid_rate = arguments["price"] / arguments["energy"]
+        if bid_rate < 0.0:
```

Comment:
same https://github.com/gridsingularity/d3a/pull/930/files#r509555830

Reply:
Hmm, here it actually makes sense to have a zero value, because the user might want to set the bid price to zero.